### ISSUE TITLE
Remove accordion anchor focus border

### ISSUE THREAD

**Issue by Martin-Luft** [valor-software/ngx-bootstrap#920]
The orginal bootstrap accordion has no anchor focus border:
http://getbootstrap.com/javascript/#collapse-example-accordion

The ng2-bootstrap accordion has an ugly anchor focus border:
http://valor-software.com/ng2-bootstrap/#/accordion

![image](https://cloud.githubusercontent.com/assets/8995517/18209376/bdd1cdac-7133-11e6-8fa7-6cfa9b7ea491.png)

@xmlking already complained about this in https://github.com/valor-software/ng2-bootstrap/issues/457

The first point is that the anchor has no `role="button"` in ng2-boostrap, the orginal bootstrap accordion has this role. The button role triggers this original bootstrap JS code:

```
// BUTTON DATA-API
// ===============

$(document)
  .on('click.bs.button.data-api', '[data-toggle^="button"]', function (e) {
    var $btn = $(e.target).closest('.btn')
    Plugin.call($btn, 'toggle')
    if (!($(e.target).is('input[type="radio"], input[type="checkbox"]'))) {
      // Prevent double click on radios, and the double selections (so cancellation) on checkboxes
      e.preventDefault()
      // The target component still receive the focus
      if ($btn.is('input,button')) $btn.trigger('focus')
      else $btn.find('input:visible,button:visible').first().trigger('focus')
    }
  })
  .on('focus.bs.button.data-api blur.bs.button.data-api', '[data-toggle^="button"]', function (e) {
    $(e.target).closest('.btn').toggleClass('focus', /^focus(in)?$/.test(e.type))
  })
```

We should add `role="button"` and set the anchor focus in `toggleOpen()` in `accordion-group.component.ts` so that the bootstrap CSS for the a href focus style matches.

**Comment by valorkin** [comment]
Awesome description! Need to implement it

**Comment by codedoge** [comment]
I'm also experiencing this problem in angular 2 and setting style classes in a component css file seems not working.

**Comment by valorkin** [comment]
BTW fixed starting v1.1.16-9
